pudb/shell.py

```diff
@@ -231,6 +231,17 @@ def run_ipython_shell(globals, locals):
     else:
         return run_ipython_shell_v11(globals, locals)
 
+
+def run_ipython_kernel(globals, locals):
+    from IPython import embed_kernel
+
+    class DummyMod(object):
+        pass
+
+    user_module = DummyMod()
+    user_module.__dict__ = globals
+    embed_kernel(module=user_module, local_ns=locals)
```

Comment:
How does qtconsole handle reconnection? (i.e. kernel ends and then comes back?) (Maybe add a note to the docs about this?)

Reply:
Reconnection may be possible, but I am not sure. The qtconsole tab connecting to the kernel will be closed after sending "quit". But the kernels spawned later in the same debugging session seem to reuse the `kernel-xxxx.json` file.

Reply:
OK, thx.